### PR TITLE
DWR-139 - adding configuration bean for YAML parsing of yaml config f…

### PR DESCRIPTION
**Pull request by rcordeau**
…iles.

---
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_common/6)

### CODE REVIEW

File: utils/build.gradle [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```
Comment (malaffoon) [inline utils/build.gradle]: this is in the parent gradle file already

File: utils/build.gradle [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+dependencies {
+    compile ('org.springframework:spring-context:+')
```
Comment (malaffoon) [inline utils/build.gradle]: this is fine for now, but i will be tempted to have something like rdw-spring-utils at some point

File: utils/build.gradle (hunk 1 above)
Comment (rcordeau) [inline utils/build.gradle]: Yeah - I realized that after pushed it up that it was redundant.

File: utils/build.gradle (hunk 2 above)
Comment (rcordeau) [inline utils/build.gradle]: I needed it because of the @Configuration annotation

**Pull request merged**